src/R/Support/Impl/Microsoft.R.Support.csproj

```diff
@@ -1,203 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I think I know why it shows. in previous PR project was removed from solution but not actually deleted. https://github.com/Microsoft/RTVS/tree/master/src/R/Support still there in master